src/DynamoCore/BuiltInAndOperators/BuiltIn.Migrations.xml

```diff
@@ -104,6 +104,10 @@
     <oldName>RemoveIfNot</oldName>
     <newName>List.RemoveIfNot</newName>
   </priorNameHint>
+  <priorNameHint>
```

Comment:
should this file have it's name changed to builtinNodes also?

Reply:
It probably does, will change it.

Reply:
Actually the migration works even without renaming this file.

Reply:
right, but shouldn't we change it just for consistency?